api/src/lib/db/projects.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { SimpleDB } from "aws-sdk";
+import * as ProjectsDb from "./projects";
+import { ProjectFactory } from "../../../spec/factories";
+
+jest.mock("aws-sdk");
+const MockSimpleDB = (SimpleDB as any) as jest.MockInstance<SimpleDB>;
+console.log = jest.fn(); // mute console.log
+
+// Create mock object that simulates the response of `new SimpleDB().someMethod(args).promise()`
+const mockAwsMethodPromiseObject = (prototype: {
```

Comment:
If I understand correctly, this can be applied to other mocked AWS objects that have a `promise` method. I think this should go to a more generic place. How about we add it somewhere in the `spec` directory?